proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -228,10 +231,10 @@ message UnbondingDelegationEntry {
   string balance = 4  [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false];
 
   // Incrementing id that uniquely identifies this entry
-  uint64 id = 5;
+  uint64 unbonding_op_id = 5;
 
   // True if this entry's unbonding has been stopped by an external module
-  bool on_hold = 6; 
+  bool unbonding_on_hold = 6; 
```

Comment:
Is this field necessary if we also have this on the `Validator` type? Or vice versa.